buyer-api/test/operations/receiveNotarizationResult.mock.js

```diff
@@ -3,17 +3,16 @@ import sinon from 'sinon';
 import test from 'ava';
 import { someNotarizationRequest } from './receiveNotarizationResult.fixture';
 
-export const getNotarizationRequest = sinon.stub();
-td.replace('../../src/facades', { getNotarizationRequest });
-export const addNotarizacionResultJob = sinon.spy();
-td.replace('../../src/queues/tranferNotarizationResultQueue', { addNotarizacionResultJob });
-
-function returnRequest(id) {
-  if (id === '1') return someNotarizationRequest;
-  return null;
-}
+const clock = sinon.useFakeTimers();
```

Comment:
why do you need `clock` here?

Reply:
This is mainly used to fix the Date value, since in the tests we only need to have a fixed one and we don't care about what specific date and time it is.